packages/notifi-dapp-example/src/components/DestinationErrorMessage.tsx

```diff
@@ -1,5 +1,4 @@
 import { NotifiTooltip } from '@notifi-network/notifi-react-card';
```

Comment:
I would not to use component in Card since it could break the style if card style possibly changes.

Reply:
I don't think we'll need a tooltip in destination section here, so I removed it for now, may need a design for where to add it in the future.